### ISSUE TITLE
"from" and "to" considered valid CSS even outside @keyframes declaration

### ISSUE THREAD

**Issue by awayken** [rossPatton/stylint#96]
"from" and "to" are valid selectors when used inside a @keyframes block, but they are not valid outside of that context. The "valid" setting does not warn when "from" and "to" are used outside of said context.

from.styl
```stylus
@keyframes rotate {
    // Valid
    from {
        transform: rotate(0deg);
    }
    // Valid
    to {
        transform: rotate(360deg);
    }
}

// Invalid
from {
    display: block;
}

// Invalid
to {
    display: block;
}
```

from.json
```json
{
    "borderNone": true,
    "brackets": false,
    "colons": false,
    "colors": false,
    "commaSpace": true,
    "commentSpace": false,
    "cssLiteral": false,
    "depthLimit": false,
    "duplicates": true,
    "efficient": true,
    "emoji": false,
    "enforceBlockStyle": false,
    "enforceVarStyle": false,
    "extendPref": false,
    "globalDupe": false,
    "indentSpaces": 4,
    "leadingZero": true,
    "maxWarnings": 10,
    "maxWarningsKill": false,
    "mixed": false,
    "namingConvention": false,
    "namingConventionStrict": false,
    "parenSpace": false,
    "placeholders": true,
    "quotePref": false,
    "semicolons": false,
    "sortOrder": false,
    "stackedProperties": true,
    "universal": true,
    "valid": true,
    "whitespace": true,
    "zeroUnits": true,
    "zIndexDuplicates": false,
    "zIndexNormalize": false
}
```

Ouput
```bash
Stylint: You're all clear!
```

I am using 26e58b76464e297d2a2c81f16e0d2823ca1e1777 on Windows 8.1.

**Comment by rossPatton** [comment]
@awayken if you pull develop, this should hopefully be fixed.

**Comment by awayken** [comment]
I can confirm that eee056f5bf304124b244eded482b07ccca2455ed allows `from` and `to` within the @keyframes context and warns as invalid outside of the @keyframes context.

**Comment by rossPatton** [comment]
:+1:

**Comment by rossPatton** [comment]
up!
